src/libraries/Microsoft.PowerFx.Core/Public/Values/CompileTimeTypeWrapperRecordValue.cs

```diff
@@ -26,10 +28,10 @@ private CompileTimeTypeWrapperRecordValue(RecordType type, RecordValue inner)
             _inner = inner;
         }
 
-        protected override bool TryGetField(FormulaType fieldType, string fieldName, out FormulaValue result)
+        protected override bool TryGetField(FormulaType fieldType, string fieldName, CancellationToken cancellationToken, out FormulaValue result)
```

Comment:
>ancellationToken cancellationToken [](http://example.com/codeflow?start=86&length=34)

do we need here? This isn't async.

Reply:
True but in Dataverse, TryGetField implementation needs it. 
Now, we could create a TryGetFieldAsync API too which would be more logical than this.